index.js

```diff
@@ -0,0 +1,219 @@
+const cards = [
+  {
+    icon: "./img.png",
+    title: "Spring boot",
+    text: `Takes an opinionated view of building 
+                Spring applications and gets you up and 
+                running as quickly as possible.`,
+  },
+  {
+    icon: "./img.png",
+    title: "Spring Framework",
+    text: `Provides core support for dependency injection, 
+                transaction management, web apps, data access, 
+                messaging, and more.`,
+  },
+  {
+    icon: "./img.png",
+    title: "Spring Data",
+    text: `Provides a consistent approach to data access – 
+                relational, non-relational, map-reduce, 
+                and beyond.`,
+  },
+  {
+    icon: "./img.png",
+    title: "Spring Cloud",
+    text: `Provides a set of tools for common patterns in distributed systems. 
+                Useful for building and deploying microservices.`,
+  },
+  {
+    icon: "./img.png",
+    title: "Spring Cloud Data Flow",
+    text: `Provides an orchestration service for composable
+                 data microservice applications on modern runtimes.`,
+  },
+  {
+    icon: "./img.png",
+    title: "Spring Security",
+    text: `Protects your application with comprehensive and 
+                extensible authentication and authorization support.`,
+  },
+];
+
+const headerItems = [
```

Comment:
Also please extract such constants to a `constants.js` file next to the main file